### PR TITLE
- Multi group membership

### PR DESCRIPTION
**Pull request by rjschwei**
+ Allow a user to be part of multiple groups. The IAM policy limit per group
    is 10. However, users may need access to more services. With this change
    a user will get associated with multiple groups.

### CODE REVIEW

**Comment by bear454** [comment]
The code changes look good; I just have one request, and one concern:

* Please add a test that uses mulitple groups ;-)

* This requires that the config change simultaneously with a code change - it would be good to have code that consumes the groups setting still accept the current key name so that the application can be updated and still run on current configs.

File: README.md [hunk 1]
```diff
@@ -212,8 +212,9 @@ AWS_ACCESS_KEY_ID = ''
 AWS_SECRET_ACCESS_KEY = ''
 AWS_ACCOUNT_ID_ALIAS = ''
 
-# Optional: Users are added to the following IAM group
-AWS_IAM_GROUP = ''
+# Optional: Users are added to the following IAM groups
+# A comma separated string of pre-created IAM groups
+AWS_IAM_GROUPS = ''
```
Comment (smarlowucf) [inline README.md]: This could just be a list instead of string which simplifies handling.

**Comment by rjschwei** [comment]
OK, switched to a list and handling the old setting.

@bear454 I need some help with the testing please

**Comment by bear454** [comment]
As discussed, we need to update the framework in order to add the tests, and all that work will happen in the new project space... so merging ;-)

**Pull request merged**